README.md

```diff
@@ -13,81 +13,41 @@ all, it's a joy to use!
-
-Support
--------
-
-Have a question? Want to chat? Run into a problem?
+Detailed instructions can be found in the [official documentation][docs].
 
- - We have a **Slack channel** at [boltcms.slack.com][bolt-slack]. To get in,
-   get yourself an invite at [slack.bolt.cm][bolt-slack-invite]
- - There’s always some people willing to chat in our **IRC channel** on
-   Freenode at **#boltcms**. No IRC client? Use [our web-based IRC client][bolt-irc]
- - There's a **Forum** at [discuss.bolt.cm][bolt-forum], where we'll gladly
-   help you sort out problems, or discuss other Bolt-related things
- - We’re pretty active on **Twitter**. Follow us, or say hello at
-   [@BoltCM][bolt-twitter]
+**NOTE:** Cloning the repository directly is only supported for development of
+the core of Bolt, see the link above for various supported options to suite
+your needs.
 
 
 Reporting issues
 ----------------
 
-When you run into an issue, be sure to provide some details on the issue.
+See our [Contributing to Bolt][contributing] guide.
 
-Please include with your report:
-- the (example) input;
-- the output you expected;
-- the output actually produced.
 
-This way we can reproduce your issue, turn it into a test and prevent the issue
-from occurring in future versions.
+Support
+-------
 
+Have a question? Want to chat? Run into a problem? See our [community][support]
+page.
 
-Extensions and Themes
----------------------
-Since Bolt 2.0, you can install extensions and themes directly from Bolt's
-interface. To browse the available extensions and themes, visit
-[market.bolt.cm][market-bolt-cm].
+---
 
--------
+[![Build Status][travis-badge]][travis] [![Scrutinizer Continuous Inspections][codeclimate-badge]][codeclimate] [![SensioLabsInsight][sensio-badge]][sensio-insight]
 
 [bolt-cm]: https://bolt.cm
-[market-bolt-cm]: https://market.bolt.cm
-[bolt-installation]: https://docs.bolt.cm/installation
-[bolt-irc]: https://bolt.cm/irc
-[bolt-slack-invite]: https://boltcms.slack.com
-[bolt-slack]: https://boltcms.slack.com
-[bolt-forum]: https://discuss.bolt.cm
-[bolt-twitter]: https://twitter.com/boltcm
-[silex]: http://silex.sensiolabs.org
-[symfony]: http://symfony.com
-[sf-components]: http://symfony.com/components
-[required-libraries]: https://docs.bolt.cm/other/credits#used-libraries-components
 [MIT-license]: http://opensource.org/licenses/mit-license.php
+[docs]: https://docs.bolt.cm/installation
+[support]: https://bolt.cm/community
 [travis]: http://travis-ci.org/bolt/bolt
 [travis-badge]: https://travis-ci.org/GawainLynch/bolt.svg?branch=release%2F3.3
 [codeclimate]: https://lima.codeclimate.com/github/bolt/bolt
 [codeclimate-badge]: https://lima.codeclimate.com/github/bolt/bolt/badges/gpa.svg
 [sensio-insight]: https://insight.sensiolabs.com/projects/4d1713e3-be44-4c2e-ad92-35f65eee6bd5
 [sensio-badge]: https://insight.sensiolabs.com/projects/4d1713e3-be44-4c2e-ad92-35f65eee6bd5/mini.png
```

Comment:
Phew … More context in my comment might have helped my avoid a shock then when you pressed the merge button and the notification popped up on my screen … as my comment was intended to mean "Did I miss something obvious" :rofl: 

Lets just rack this up to #teamwork :smile: